app/src/main/res/values-it/strings.xml

```diff
@@ -1,282 +1,291 @@
+    <string name="home_menu">Menù</string>
+    <string name="home_post_view_mode">Formato post</string>
     <string name="home_profile">Profilo</string>
-    <string name="home_inbox">Posta in Arrivo</string>
+    <string name="home_refresh">Aggiorna</string>
+    <string name="home_saved">Salvati</string>
     <string name="home_settings">Impostazioni</string>
+    <string name="home_sign_out">Esci</string>
+    <string name="home_site_info">Info istanza</string>
+    <string name="home_subscribed">Iscrizioni</string>
     <string name="home_subscriptions">Iscrizioni</string>
-    <string name="home_add_account">Aggiungi Profilo</string>
     <string name="home_switch_to">Usa %1$s/%2$s</string>
-    <string name="home_sign_out">Esci</string>
-    <string name="home_menu">Menù</string>
-    <string name="home_refresh">Aggiorna</string>
-    <string name="home_post_view_mode">Formato Post</string>
-    <string name="home_site_info">Info Istanza</string>
     <string name="inbox_back">Indietro</string>
-    <string name="inbox_inbox">Posta In Arrivo</string>
-    <string name="login_password">Password</string>
+    <string name="inbox_filter">Seleziona filtro per la posta in arrivo</string>
+    <string name="inbox_inbox">Posta in arrivo</string>
+    <string name="inbox_markAllRead">Segna tutti come già letti</string>
+    <string name="input_fields_cancel">Annulla</string>
+    <string name="input_fields_insert_link">Inserisci link</string>
+    <string name="input_fields_link">Link</string>
+    <string name="input_fields_ok">OK</string>
+    <string name="input_fields_text">Testo</string>
+    <string name="lessOptions">Mostra meno opzioni</string>
+    <string name="login_back">Indietro</string>
+    <string name="login_email_or_username">Email o nome utente</string>
     <string name="login_instance">Istanza</string>
     <string name="login_instance_placeholder">ad es.: lemmy.ml</string>
-    <string name="login_email_or_username">Email o Nome Utente</string>
     <string name="login_login">Login</string>
-    <string name="login_back">Indietro</string>
+    <string name="login_password">Password</string>
+    <string name="login_view_model_incorrect_login">Login errato</string>
+    <string name="login_view_model_is_not_a_lemmy_instance">%1$s non è un\'istanza Lemmy</string>
+    <string name="look_and_feel_activity_show_content_for_collapsed_comments">Mostra contenuto per commenti minimizzati</string>
+    <string name="look_and_feel_font_size">Dimensione carattere: %1$s</string>
+    <string name="look_and_feel_look_and_feel">Aspetto</string>
+    <string name="look_and_feel_post_view">Formato post</string>
+    <string name="look_and_feel_post_view_card">Scheda</string>
+    <string name="look_and_feel_post_view_list">Lista</string>
+    <string name="look_and_feel_post_view_small_card">Scheda piccola</string>
+    <string name="look_and_feel_show_action_bar_for_comments">Mostra barra delle azioni per i commenti</string>
```

Comment:
"Mostra sempre la barra delle azioni per i commenti"?

If this setting isn't active you can still see the action bar, you just have to long-press the comment body... Can we communicate this better without too long a sentence?